var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -181,9 +184,17 @@ def test(self):
             args.append('HEADER=' + self.prefix.include)
         args.append('superlu')
 
-        test_dir = join_path(self.install_test_root, self.examples_src_dir)
+        test_dir = join_path(self.test_suite.current_test_cache_dir,
+                             self.examples_src_dir)
+
+        if not os.path.exists(test_dir):
+            print('Skipping superlu test')
+            return
+
         with working_dir(test_dir, create=False):
             make(*args, parallel=False)
-            self.run_test('./superlu', purpose='Smoke test for superlu',
+            self.run_test('./superlu',
+                          options=['-I{0}'.format(join_path(self.prefix, 'include'))],
```

Comment:
Recommend using the approach at line 184:

```suggestion
                          options=['-I{0}'.format(self.prefix.include)],
```